test/kubemark/start-kubemark.sh

```diff
@@ -68,11 +68,52 @@ gcloud compute firewall-rules create "${INSTANCE_PREFIX}-kubemark-master-https"
     --network "${NETWORK}" \
     --source-ranges "0.0.0.0/0" \
     --target-tags "${MASTER_TAG}" \
-    --allow "tcp:443" || true
+    --allow "tcp:443"
 
 MASTER_IP=$(gcloud compute instances describe ${MASTER_NAME} \
   --zone="${ZONE}" --project="${PROJECT}" | grep natIP: | cut -f2 -d":" | sed "s/ //g")
 
+if [ "${SEPARATE_EVENT_MACHINE:-false}" == "true" ]; then
+  EVENT_STORE_NAME="${INSTANCE_PREFIX}-event-store"
+  gcloud compute disks create "${EVENT_STORE_NAME}-pd" \
+      --project "${PROJECT}" \
```

Comment:
Can we factor out all of the common gcloud command line arguments into one central place, perhaps something like $GCLOUD_COMMON_ARGS (e.g. --project, --zone, --quiet)